packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/OverflowMenuDemo/OverflowMenuDemo.tsx

```diff
@@ -87,7 +87,7 @@ export class OverflowMenuDemo extends React.Component {
     });
   };
 
-  onAdditionalOptionsSelect = event => {
+  onAdditionalOptionsSelect = () => {
```

Comment:
Should we restore the `event` props in the functions above? Perhaps just rename it as _event to avoid the warning?